packages/start-slicemachine/src/lib/safelyExecute.ts

```diff
@@ -0,0 +1,9 @@
+export async function safelyExecute<T>(
```

Comment:
This is pedantic, but `safelyExecute` implies the function will execute something. Instead, this functions awaits something that was already executed in a Promise.

How about `safelyAwait` or `safelyTry`?

Or change `operation` to a function that executes the function when `safetyExecute` is called:

```ts
safelyExecute(() => this._sliceMachineManager.simulator.getLocalSliceSimulatorURL())
```

Reply:
100% right, I like your second proposal 🙂